posthog/api/capture.py

```diff
@@ -39,12 +40,21 @@ def _load_data(request) -> Optional[Union[Dict, List]]:
     if not data:
         return None
 
+    # add the data in sentry's scope in case there's an exception
+    with push_scope() as scope:
+        scope.set_context("data", data)
+
     #  Is it plain json?
     try:
         data = json.loads(data)
     except json.JSONDecodeError:
         # if not, it's probably base64 encoded from other libraries
-        data = json.loads(base64.b64decode(data + "===").decode('utf8', 'surrogatepass').encode('utf-16', 'surrogatepass'))
+        data = base64.b64decode(data + "===")
+        try:
+            data = data.decode('utf8', 'surrogatepass').encode('utf-16', 'surrogatepass')
```

Comment:
We have a test case for this one, maybe worth having a test case for 'utf8' 'ignore' too?